pageserve/static/style.css

```diff
@@ -0,0 +1,48 @@
+/* Add a black background color to the top navigation */
+.topnav {
+  background-color: #333;
+  overflow: hidden;
+}
+
+/* Style the links inside the navigation bar */
+.topnav .navtab {
+  float: left;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.topnav .navbtn:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/* Add a color to the active/current link */
+.topnav .navbtn.active {
+  background-color: #4CAF50;
+  color: white;
+}
+
+#user_nav {
+  float: right;
+}
+
+#sign_out_button {
+  border-left: 1px solid white;
+}
+
+#sign_in_button {
+  padding: 4% 8px;    /* TODO: vertically center sign-in button */
```

Comment:
https://css-tricks.com/centering-css-complete-guide/

Reply:
Flexbox solved this in 91c9612